apps/api_web/lib/api_web/controllers/alert_controller.ex

```diff
@@ -122,7 +122,7 @@ defmodule ApiWeb.AlertController do
       |> apply_filters()
       |> case do
         list when is_list(list) ->
-          State.all(list, Params.filter_opts(params, @pagination_opts))
+          State.all(list, Params.filter_opts(params, @pagination_opts, conn))
```

Comment:
`params` are a part of `conn`: do you need both?

Reply:
Are they the same params? I see a lot of examples of functions which take `params` as well as `conn`; e.g. `index_data/2` in each controller.

Separately, is there any cost to passing the whole `conn` around? I'm only using the `api_version`, so I could also rewrite to just pass the version to `Params.filter_opts` if that would be preferable.

Reply:
It's all in the same process, so I don't think there's any issue with passing the conn around. We can leave it as a separate parameter for now.